### PR TITLE
Add Zone as Parameter to CRUD operations

### PR DESCRIPTION
**Pull request by NavisJ**
# Description
Import of Nfs Export Settings with parameter does not function as expected.
Need to add parameter to other function as well, which will be affected by import.

# ISSUE TYPE
- Bugfix Pull Request

##### RESOURCE OR DATASOURCE NAME
NFS Export Settings

# Checklist:

- [x] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [x] I have verified that new and existing unit tests pass locally with my changes
- [x] I have not allowed coverage numbers to degenerate
- [x] I have maintained at least 80% code coverage
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have maintained backward compatibility

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Please also list any relevant details for your test configuration

- [x] Unit tests
- [x] Acceptance tests

### CODE REVIEW

**Comment by NavisJ** [comment]
run e2e test

**Comment by NavisJ** [comment]
run e2e test

**Review by baoy1 — APPROVED** [review]

File: powerscale/provider/nfs_export_settings_resource.go [hunk 1]
```diff
@@ -940,7 +940,7 @@ func (r *NfsExportSettingsResource) Create(ctx context.Context, req resource.Cre
 		return
 	}
 
-	settings, err := helper.GetNfsExportSettings(ctx, r.client)
+	settings, err := helper.GetNfsExportSettingsByZone(ctx, r.client, plan.Zone.ValueString())
```
[the remark this answers is not in the document]
Comment (NavisJ) [reply powerscale/provider/nfs_export_settings_resource.go:943]: Added sample in tf file.

**Comment by NavisJ** [comment]
run e2e test

**Review by taohe1012 — APPROVED** [review]

**Comment by NavisJ** [comment]
run e2e test

**Comment by forrestxia** [comment]
run e2e test

**Comment by taohe1012** [comment]
run e2e test

**Comment by taohe1012** [comment]
run e2e test

**Comment by forrestxia** [comment]
run e2e test

**Comment by forrestxia** [comment]
run e2e test

**Comment by NavisJ** [comment]
run e2e test

**Pull request merged**